docs/adr/0011-operators.md

```diff
@@ -0,0 +1,153 @@
+2. When deployed it generates `ClusterRole` and `ClusterRoleBinding` and aggregation rules.
+3. It can specify all namespaces visibility for an operator.
+4. Dy default project admins do not have create/edit rights to `OperatorGroup`s (due to points 2 and 3), so only cluster-admins can manage _Operator groups_.
+
+There is a number [of possible conflicts](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/operatorgroups.md#what-can-go-wrong) that are a direct consequence of an `OperatorGroup` deployment. These resources needs to be treated carefully and therefore it is required for them to be placed and maintained at `cluster-scope` application level.
+
+Centralized `Operator Group` management ensures we won't be deploying the same namespaced operator while we already have another instance with all namespace visibility, etc.
+
+According to the upstream documentation on [`OperatorGroup` target namespace selection][] we enforce and support only these options for target namespace specification in _Operator groups_ specification:
+
+1. A global, cluster-wide operator with visibility to all namespaces, that means the `.spec` section of the manifest is blank.
+2. A single namespace operator. Resource manifest specifies `.spec.targetNamespaces: [ SINGLE_MEMBER_NAMESPACE ]`. This namespace has to be deployed via `cluster-scope` application as well.
+
+The `OperatorGroup` resource manifest is expected to be deployed to the same namespace where the corresponding `Subscription` is located.
+
+### Direct operator deployment
+
+Some operators are not deployed via the OLM. That means, their manifests require manual deployment of `CustomResourceDefinition`s, `ClusterRole`s etc. We require these kind of resources to be deployed via `cluster-scope` application.
+
+In contrast to that, all the other resources required for a non-OLM operator deployment, like `Deployment`, `Route` etc. are treated as any other namespace scoped application and all the manifests required for deployment are expected to be located as a separate application in the `operate-first/apps` repository as a root folder.
+
+### Meta-operators
+
+This is a special kind of operator. A meta-operator in this context is understood to be any operator allowing deployment of other operators at cluster scope. Our intention is to limit the possibilities of such operator to be able to interfere with our cluster management in `cluster-scope` application. We expect from meta-operators to provide options to either:
+
+- Limit their permissions via operator configuration, so they are not able to deploy resources requiring cluster-admin permissions anymore.
+- Limit the create/edit access permission to the custom resources they create, in order to restrict normal project users or project admins to escalate their privileges.
```

Comment:
restrict ... to escalate. That sounds strange. Maybe: prevent... from escalating?